Update to import data into Advanced eDiscovery (#3020)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/import-non-office-365-data-into-advanced-ediscovery.md

1. As an eDiscovery Manager or eDiscovery Administrator, open **eDiscovery**, and open the case that the non-Office 365 data will be uploaded to. If you need to create a case, see [Manage eDiscovery cases in the Security &amp; Compliance Center](ediscovery-cases.md)
    
2. Click **Switch to Advanced eDiscovery**

> 3. Select **Review Sets** from the menu

Reviewer: ```suggestion
3. Select **Review Sets** from the menu.
```

> 4. Select and existing Review Set or click on **Add Review Set**

Reviewer: ```suggestion
4. Select an existing Review Set or choose **Add Review Set**.
```

> 5. Select **Manage review set**

Reviewer: ```suggestion
5. Select **Manage review set**.
```

> 6. In the Non-Office 365 data card, click on **View Uploads**

Reviewer: ```suggestion
6. In the Non-Office 365 data card, select **View Uploads**.
```

> 7. Click on **Upload files** to start the file upload wizard

Reviewer: ```suggestion
7. Choose **Upload files** to start the file upload wizard.
```

> 8. The first tab is the **1. Prepare step**. Click on **Next: Upload files**

Reviewer: ```suggestion
8. The first tab is **1. Prepare step**. Select **Next: Upload files**.
```

> 9. On the **2. Upload files** tab you will be promoted to download AzCopy.exe if you have not and then provide the path to the file location.  For example, `C:\Upload`.  This will give you the command to execute for AzCopy.exe, for example using `C:\Upload` you would see:

Reviewer: ```suggestion
9. On the **2. Upload files** tab you will be prompted to download AzCopy.exe if you have not done so already, and then to provide the path to the file location. For example, `C:\Upload`  will give you the command to execute AzCopy.exe. Using `C:\Upload`, you will see:
```

  `"%ProgramFiles(x86)%\Microsoft SDKs\Azure\AzCopy\AzCopy.exe" /Source:"c:\upload" /Dest:"https://spnam03salinkexternal003.blob.core.windows.net/16d13440-a6a4-4bc5-a82b-10ac9cfe9d7c-1601401811-externalstore?sv=2017-07-29&sr=c&si=ExternalStore63%7C0&sig=9Dq5v20TwkxByYDHhIEx%2FHSLlmlqUjY0njkJyTO0zGA%3D" /s`
  

> 10. Open a command prompt window and execute the AzCopy.exe command to import the data into Azure.  Once it has loaded all of the data then click on **Next: Process files**

Reviewer: ```suggestion
10. Open a command prompt window and execute the AzCopy.exe command to import the data into Azure. Once it has loaded all of the data, select **Next: Process files**.
```

> 11. The next tab is **3. Process files** where you will see the custodians that had data associated with them and will show you the progress of the data being imported.

Reviewer: ```suggestion
11. The next tab is **3. Process files** where you will see the custodians that have data associated with them and will also show you the progress of the data being imported.
```

>     For more information on Azcopy syntax see, [Transfer data with the AzCopy on Windows](https://docs.microsoft.com/azure/storage/common/storage-use-azcopy) . 

Reviewer: ```suggestion
    For more information on Azcopy syntax, see [Transfer data with the AzCopy on Windows](https://docs.microsoft.com/azure/storage/common/storage-use-azcopy). 
```

>     For more details on Advanced eDiscovery Processing see, [Run the Process module and load data in Advanced eDiscovery](run-the-process-module-and-load-data-in-advanced-ediscovery.md) 

Reviewer: ```suggestion
    For more details on Advanced eDiscovery Processing, see [Run the Process module and load data in Advanced eDiscovery (classic)](run-the-process-module-and-load-data-in-advanced-ediscovery.md). 
```
